scripts/stop-local-env.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -ue
+
+reg_network='kind'
+
+kind_registry='kind-registry'
+kind_control_plane='kind-control-plane'
+BITBUCKET_POSTGRES_CONTAINER_NAME="bitbucket-postgres-test"
+BITBUCKET_SERVER_CONTAINER_NAME="bitbucket-server-test"
+NEXUS_CONTAINER_NAME="nexustest"
+SQ_CONTAINER_NAME="sonarqubetest"
+
+container_names_in_stop_order=( "$SQ_CONTAINER_NAME" "$NEXUS_CONTAINER_NAME" "$BITBUCKET_SERVER_CONTAINER_NAME" "$BITBUCKET_POSTGRES_CONTAINER_NAME" "$kind_control_plane" "$kind_registry" ) 
+
+for cn in "${container_names_in_stop_order[@]}"; do
+    echo docker stop "$cn"
+    docker stop "$cn"
+done
+
+# echo docker stop "$(docker ps -qf "network=$reg_network")"
```

Comment:
Why did you end up not doing this? Because it might be too broad?

Reply:
Mostly because during restart I wanted to start containers in a certain order and thought a symmetry might be better. 
Also thought about having a `make clean-local-env' target which would `docker rm` the containers, but then `docker ps` can no longer report the containers anymore.